andela/cyclops-backend
# Complete sequelize setup using sequelize-cli

**What does this PR do?**

Complete sequelize setup using sequelize-cli

**How should this be manually tested?**

- Clone repo and run `npm install`
- Add dotenv package and add the database variable to the .env file, see config/database.json
- Run `/node_modules/.bin/sequelize db:migrate` to migrate dummy user migration

**Any background context you want to provide?**

- Sequelize ORM is required in the project specification to be used with Postgresql.

**What are the relevant stories?**

[#167728004](https://www.pivotaltracker.com/n/projects/2381984/stories/167728004)

**Screenshots (if appropriate)**

- None
